src/test/java/com/icthh/xm/uaa/service/TenantRoleServiceUnitTest.java

```diff
@@ -145,16 +141,26 @@ public void testGetRolePermissions() {
 
         when(tenantConfigRepository.getConfigFullPath(XM_TENANT, PERMISSIONS_PATH))
             .thenReturn(readConfigFile("/config/tenants/XM/permissions.yml"));
-        assertFalse(tenantRoleService.getRolePermissions(roleKey).isEmpty());
-        assertEquals("ROLE_ADMIN", tenantRoleService.getRolePermissions(roleKey).get(0).getRoleKey());
-        assertEquals("ATTACHMENT.CREATE", tenantRoleService.getRolePermissions(roleKey).get(0).getPrivilegeKey());
+        List<PermissionDTO> rolePermissions = tenantRoleService.getRolePermissions(roleKey);
```

Comment:
pls use separate  file with permission for testGetExistingAccount, and don't add not needed changes in existing test

Reply:
added another test